src/einsteinpy/plotting/geodesics/geodesic_plotter.py

```diff
@@ -0,0 +1,22 @@
+import random
+
+from .interactivegeodesic_plotter import InteractiveGeodesicPlotter
```

Comment:
No relative imports

Reply:
But if we import a plotting module from another plotting module that raises an error due to circular import.

Reply:
But if we try to import a plotting module from another plotting module it would give an error due to circular import. We have to give a relative path in either __init__.py or core.py.